packages/backend/src/peripherals/database/migrations/035_tipped_transaction_count_view_zksync.ts

```diff
@@ -0,0 +1,38 @@
+/*
+                      ====== IMPORTANT NOTICE ======
+
+DO NOT EDIT OR RENAME THIS FILE
+
+This is a migration file. Once created the file should not be renamed or edited,
+because migrations are only run once on the production server. 
+
+If you find that something was incorrectly set up in the `up` function you
+should create a new migration file that fixes the issue.
+
+*/
+
+import { ProjectId } from '@l2beat/types'
+import { Knex } from 'knex'
+
+import * as oldView from './032_transaction_count_view_zksync'
+
+export async function up(knex: Knex) {
+  await oldView.down(knex)
+  await knex.schema.raw(
+    `
+  CREATE MATERIALIZED VIEW transactions.zksync_count_view AS
+    SELECT
+      date_trunc('day', zksync.unix_timestamp) unix_timestamp,
+      count(*) count
+    FROM transactions.zksync zksync
+    INNER JOIN transactions.block_tip tip ON tip.project_id = '${ProjectId.ZKSYNC.toString()}'
```

Comment:
Please use bindings

Reply:
I wanted to use that, but `knex.schema.raw` does not accept bindings. Now that I checked it's just types (https://github.com/knex/knex/issues/4069) - will update